spring-kafka-docs/src/main/asciidoc/retrytopic.adoc

```diff
@@ -48,6 +48,11 @@ NOTE: You can have separate `ListenerContainerFactory` instances for the main an
 
 ==== Configuration
 
+Since 2.9, the `@EnableKafkaRetryTopic` annotation should be used in a `@Configuration` annotated class instead of the regular `@EnableKafka` annotation, which it contains.
```

Comment:
That's not good say like this: brings an impression that we must not use an `@EnableKafka` anymore. 
And you even didn't introduce what is `@EnableKafkaRetryTopic`. The side effect with the `@EnableKafka` could be explained in the end of this paragraph. Probably in the simple "NOTE: When `@EnableKafkaRetryTopic` is used, the `@EnableKafka` can be omitted: the former is meta-annotated with the second."